python/cudf/cudf/core/_internals/expressions.py

```diff
@@ -56,6 +56,7 @@
     # https://pandas.pydata.org/pandas-docs/stable/user_guide/enhancingperf.html#expression-evaluation-via-eval  # noqa: E501
     # that we don't support yet:
     # expm1, log1p, arctan2 and log10.
+    "isnull": ASTOperator.IS_NULL,
```

Comment:
Question for reviewers: since this isn't an operator/function supported by pandas's eval function, we have more flexibility in the name. Do we like `isnull`, for consistency with Series/DataFrame.isnull? I do, just want to gauge opinions during review.

Reply:
pandas `pd.isnull` is why `isnull` was choosen here. But `pd.isnull` is same as `pd.isna`.
https://github.com/pandas-dev/pandas/blob/779a0ffb14ee3237fdb7b9d53ef5b7b0b8ebe473/pandas/core/dtypes/missing.py#L178

@jlowe @bdice @GregoryKimball for opinions.

Reply:
added `isna` as operator.